cancel-orders-mainnet.js

```diff
@@ -5,12 +5,12 @@ import fetch from 'node-fetch';
 // ***** Need to update PRIVATE_KEY, market id and username  ********
 // To export private key from your wallet, follow:
 // https://help.proton.org/hc/en-us/articles/4410313687703-How-do-I-backup-my-private-key-in-the-WebAuth-Wallet-
-const PRIVATE_KEY = 'private_key';
+const PRIVATE_KEY = '5JsYG1B6UJjBDNaJWrQb9kjEiF6sw6dHZVx59AvRM84h5btHkVr';
```

Comment:
is this a testnet key?

Reply:
nope lol. caught it and changed asap luckily